text/0003-multisig-minting/0003-multisig-minting.md

```diff
@@ -46,26 +48,5 @@ With implementation of the dependent RFCs, the minter is free to modify the time
-The cold wallet will consist of 2 *move keys*.
-The hot wallet will consist of 1 *move key* and 2 *mint keys*.
-The pool operator will be given 1 *mint key*.
-The hot wallet fully signs coinstake transactions every 30 days and gives them to the pool operator to build blocks, such that they can sign the coinbase transaction using the third key.
-The coins cannot be moved without at least one cold wallet key.
-The benefit of this arrangement is that switching pool operators does not require moving the coins or accessing the cold wallet.
-Indeed, the *mint key* given to the pool operator has so little power that it can be given out freely to pool operators.
-
-When switching to a new pool without moving the coins, the old pool operator would still have signed coinstake transactions that it can mint.
-By providing the new pool operator with signed coinstake transactions from the same outputs, the new pool operator would have the power to cause blocks submitted by the old pool operator to be banned.
-
-This is to be contrasted with a 1-of-2 address with a *mint key* held by a pool operator or hot wallet and a *move key* held in cold storage, which is the quintessential *cold minting* and does not require splitting the coinstake at all.
-However, when switching to a different pool operator or when the hot wallet is compromised, the coins must be moved and the coinage burned to prevent the unwanted use of that coinage.
-The 2-of-4+ multisignature address with *cold minting* allows for varying degrees of security, depending on how many of the keys are cold or hot (the hot wallet and the pool operator need at least one *mint key* each, and there have to be at least 2 *move keys*).
-The driving benefit of a setup like this is the flexibility to change pool operators without accessing cold keys or consuming coinage.
+* The convention could be chosen that the mint reward must be sent to the minting address when multisignature minting. However, by giving the minter the freedom to choose the destination of the mint reward, we allow for socialization of rewards in a mint pool setting.
+* Cold minting and multisig minting can be combined to provide unparalleled security and utility.  To illustrate, I will use the concept of mint keys and an address with 1 move key and 2-of-3 mint keys.  The 1 move key is cold storage and kept off-line at all times. The hot wallet will hold 2 of the mint keys and will periodically sign coinstake transactions once every 30 days (such that even the hot wallet can be airgapped). The final mint key is given to the pool operator, who has the authority to take the mint reward and socialize it with others in the pool. Combining these two methods and developing the user interface around them may drastically alter the way the average user mints, without affecting the current single-signature method.
```

Comment:
With the coinbase and coinstake transaction splitted, the coinstake (coinburn) transaction behaves just like any regular transaction.
So special rules would be required to allow it to be moved using "mint keys", but I guess that's the entire point of sigmike's cold minting.

Reply:
With my currently proposed implementation for cold minting the pool operator would indeed not be allowed to socialize the reward. That's a design choice (https://talk.peercoin.net/t/cold-storage-minting-proposal/2336/125) that can be changed. We can also change the design to let the owner choose when he creates the address whether the minter is allowed to get the reward or not.

I still think it's better to prevent pools, but that's only my personal opinion.